src/variations.json

```diff
@@ -9,7 +9,8 @@
     "weight": 1,
     "prefs": {
       "urlclassifier.trackingAnnotationTable": "test-track-simple,ads-track-digest256,social-track-digest256,analytics-track-digest256,content-track-digest256",
-      "network.cookie.cookieBehavior": 4
+      "network.cookie.cookieBehavior": 4,
+      "privacy.annotate_channels.strict_list.enabled": true
```

Comment:
BTW, should we be setting these prefs to the values we want to control in the control branch as well for scientific rigor?